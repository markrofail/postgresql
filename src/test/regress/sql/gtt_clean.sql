

reset search_path;

drop schema gtt cascade;

