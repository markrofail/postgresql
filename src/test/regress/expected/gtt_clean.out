reset search_path;
drop schema gtt cascade;
NOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to table gtt.gtt1
drop cascades to table gtt.gtt2
drop cascades to table gtt.gtt3
drop cascades to table gtt.gtt_t_kenyon
